### PR TITLE
Added Error Handling in HealthcareApis and fixed some minor issues in Set-AzHealthcareApisService and documents

### PR DESCRIPTION
**Pull request by poadhika**
## Description
This PR addresses the following:
* Added Error Handling in all cmdlets
* Fixed few typos
* Enable Set-AzHealthcareApisService to allow updating tags.

## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Azure.Commands.HealthcareApis.Properties;
```
Comment (cormacpayne) [inline src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs]: @poadhika please add a license header to this file

File: src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.Common
+{
+    public static class HealthcareApisArgumentValidator
+    {
+
```
Comment (cormacpayne) [inline src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs]: @poadhika nit: remove this empty line

File: src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        public static bool ValidateObjectId(string AccessPolicyObjectId)
```
Comment (cormacpayne) [inline src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs]: @poadhika nit: please use camel case for variables: `AccessPolicyObjectId` --> `accessPolicyObjectId`

File: src/HealthcareApis/HealthcareApis/HealthcareApis/SetAzureRmHealthcareApisService.cs [hunk 4]
```diff
@@ -257,10 +263,24 @@ private ServicesDescription GenerateServiceDescription(ServicesDescription healt
...
+                Tags = GetTags(healthcareApisAccount)
             };
         }
 
+        private IDictionary<string, string> GetTags(ServicesDescription healthcareApisAccount)
+        {
+            if(this.Tag!=null && this.Tag.Count > 0)
+            {
+                Dictionary<string, string> dict = new Dictionary<string, string>();
+                foreach (DictionaryEntry kvp in this.Tag)
```
Comment (cormacpayne) [inline src/HealthcareApis/HealthcareApis/HealthcareApis/SetAzureRmHealthcareApisService.cs]: @poadhika nit: please add the missing curly braces

```cs
foreach (DictionaryEntry kvp in this.Tag)
{
    dict.Add((string)kvp.Key, (string)kvp.Value);
}
```

**Pull request merged**